src/main/java/Event.java

```diff
@@ -0,0 +1,31 @@
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+
+    private LocalDateTime at;
```

Comment:
I like the use of "at" and "by" to denote the time for the different event types!